test/web-auth/captcha.test.js

```diff
@@ -531,51 +752,75 @@ describe('passwordless captcha rendering', function () {
     });
 
     it('should set the image tag', function () {
-      expect(element.innerHTML).to.equal(`custom template: ${challenges[0].image}`);
+      expect(element.innerHTML).to.equal(
+        `custom template: ${challenges[0].image}`
+      );
     });
   });
 
   const RECAPTCHA_V2_PROVIDER = 'recaptcha_v2';
   const RECAPTCHA_ENTERPRISE_PROVIDER = 'recaptcha_enterprise';
   const HCAPTCHA_PROVIDER = 'hcaptcha';
   const FRIENDLY_CAPTCHA_PROVIDER = 'friendly_captcha';
-
-  [RECAPTCHA_V2_PROVIDER,RECAPTCHA_ENTERPRISE_PROVIDER, HCAPTCHA_PROVIDER, FRIENDLY_CAPTCHA_PROVIDER].forEach(provider => {
+  const ARKOSE_PROVIDER = 'arkose';
+
+  [
+    RECAPTCHA_V2_PROVIDER,
+    RECAPTCHA_ENTERPRISE_PROVIDER,
+    HCAPTCHA_PROVIDER,
+    FRIENDLY_CAPTCHA_PROVIDER
+  ].forEach(provider => {
```

Comment:
Should we add ARKOSE_PROVIDER in this array?

Reply:
No I created a separate describe block for Arkose. This array is running a set of tests for all the captcha providers (except Arkose). But in the situation of Arkose, the setup is different and we need more tests and I didn't want to have all these conditionals in the existing tests for Arkose.

Reply:
Should we get rid of the const added above it for arkose?

Reply:
The `const` is used in the Arkose tests. I just put it there to group all the consts together but happy to move it if you think so.

Reply:
No let's keep it. Thanks for clarifying!